app/serializers/program_session_serializer.rb

```diff
@@ -1,5 +1,5 @@
 class ProgramSessionSerializer < ActiveModel::Serializer
-  attributes :title, :abstract, :format, :track, :id#, :review_tags, :custom_fields, :video_url, :slides_url
+  attributes :title, :abstract, :format, :track, :id, :review_tags, :custom_fields, :video_url, :slides_url
```

Comment:
We only want video url and slides url, not review tags or custom fields as they don't exist on ProgramSessions.  Go ahead and remove those.